builder/main.py

```
@@ -343,33 +426,75 @@ def check_lib_archive_exists():
     )
 )
 
+# Load framework-specific configuration
 if not env.get("PIOFRAMEWORK"):
     env.SConscript("frameworks/_bare.py", exports="env")
 
-
 def firmware_metrics(target, source, env):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)